src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/migrations/add/AddMigrationDialogWrapper.kt

```diff
@@ -124,4 +139,11 @@ class AddMigrationDialogWrapper(
         migrationNameTextField.text = migrationName
         migrationNameTextField.document.addDocumentListener(migrationNameChangedListener)
     }
+
+    private fun getCurrentMigrationProjectFolder(): String? {
+        val currentMigrationsProject = commonOptions.migrationsProject?.data?.fullPath
+        val migrationsProjectFolderPath = File(currentMigrationsProject!!).parentFile.path
```

Comment:
Please cover case with nullable migration folder (if the solution contains no supported Migrations project)